backend/modules/Pantheon/Console/Commands/PantheonSyncCommand.php

```
@@ -0,0 +1,209 @@
+        $only = $this->option('only');
+        $selected = $only ? array_intersect_key(self::SYNCS, [$only => true]) : self::SYNCS;
+
+        if ($selected === []) {
+            $this->error("Unknown sync '{$only}'. Available: ".implode(', ', array_keys(self::SYNCS)));
+
+            return self::FAILURE;
+        }
+
+        // withoutGlobalScopes: this is the one query that must see every tenant,
+        // because it is what tells us which tenants to iterate.
+        $configs = IntegrationConfig::withoutGlobalScopes()
```

PHPStan (2.2.7)

[error] Call to an undefined static method App\Models\IntegrationConfig::withoutGlobalScopes().


(staticMethod.notFound)